frontend/src/pages/LandingPage/LandingPage.tsx

```diff
@@ -1,6 +1,199 @@
+/* eslint-disable react/no-unescaped-entities */
 import React from 'react';
+import drag from './Assets/img/drag-and-drop.png';
+import flag from './Assets/img/flag.png';
+import padlock from './Assets/img/open-padlock.png';
+import view from './Assets/img/view.png';
+import S from './styles';
 
-// TODO: Setup landing page
-const LandingPage = () => <div>LandingPage</div>;
+const LandingPage = () => (
+  <S.LandingPageContainer>
+    <div>
+      <svg
```

Comment:
I think I said it was fine to insert the svg inline 😅. But can we instead store the svg into an assets folder?